README.rst

```diff
@@ -67,6 +67,8 @@ Config
 Put this into ``/home/user/.config/radicale/config`` (replace ``/home/user`` by your ``$HOME``).
 The ``remind_file``, ``abook_file`` and ``task_folder`` are optional, and can be removed if not used.
 
+It is required that the ``filesystem_folder`` be set to ``/home``.  
```

Comment:
maybe rather something like:
`filesystem_folder` should be everything up to your user name?